easybuild/easyconfigs/r/R/R-3.3.3-foss-2016b.eb

```diff
@@ -0,0 +1,500 @@
+name = 'R'
+version = '3.3.3'
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://cran.us.r-project.org/src/base/R-%(version_major)s']
+
+preconfigopts = 'BLAS_LIBS="$LIBBLAS" LAPACK_LIBS="$LIBLAPACK"'
+configopts = "--with-lapack --with-blas --with-pic --enable-threads --with-x=no --enable-R-shlib"
+# some recommended packages may fail in a parallel build (e.g. Matrix), and we're installing them anyway below
+configopts += " --with-recommended-packages=no"
+
+dependencies = [
+    ('libreadline', '6.3'),
+    ('ncurses', '6.0'),
+    ('bzip2', '1.0.6'),
+    ('XZ', '5.2.2'),
+    ('zlib', '1.2.8'),
+    ('SQLite', '3.13.0'),
+    ('PCRE', '8.39'),
+    ('libpng', '1.6.24'),  # for plotting in R
+    ('libjpeg-turbo', '1.5.0'),  # for plottting in R
+    ('LibTIFF', '4.0.6'),
+    ('Java', '1.8.0_92', '', True),  # Java bindings are built if Java is found, might as well provide it
+    ('Tcl', '8.6.5'),  # for tcltk
+    ('Tk', '8.6.5'),  # for tcltk
+    ('cURL', '7.49.1'),  # for RCurl
+    ('libxml2', '2.9.4'),  # for XML
+    ('GDAL', '2.1.0'),  # for rgdal
+    ('PROJ', '4.9.2'),  # for rgdal
+    ('GMP', '6.1.1'),  # for igraph
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.0.2h'),
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
+
+name_tmpl = '%(name)s_%(version)s.tar.gz'
+ext_options = {
+    'source_urls': [
+        'http://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'http://cran.r-project.org/src/contrib/',  # current version of packages
+        'http://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': name_tmpl,
+}
+
+
+# !! order of packages is important !!
+# packages updated on Mar 9th 2017
+exts_list = [
+    # default libraries, only here to sanity check their presence
+    'base',
+    'datasets',
+    'graphics',
+    'grDevices',
+    'grid',
+    'methods',
+    'splines',
+    'stats',
+    'stats4',
+    'tools',
+    'utils',
+    # non-standard libraries, should be specified with fixed versions!
+    # the commented-out extensions need either X, cairo or mesa
```

Comment:
@wpoely86 What's the point of adding an extra R easyconfig with the same list of extensions as the existing `R-3.3.3-foss-2016b-X11-20160819.eb`, where a couple ones are commented out because X11 & co are not included as dependencies?